openmetadata-ui/src/main/resources/ui/playwright/utils/advancedSearch.ts

```diff
@@ -182,7 +182,7 @@ export const fillRule = async (
       }
 
       await page
-        .locator(`.ant-select-dropdown [title="${searchData}"]`)
+        .locator(`.ant-select-dropdown:visible [title="${searchData}"]`)
```

Comment:
https://github.com/open-metadata/OpenMetadata/blob/35bed7138df2d015082b8015466708c2a22649d0/openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/ServiceBaseClass.ts#L315 
Consider above approach to point to latest dropdown options